gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/entities.proto

```diff
@@ -10,6 +10,7 @@ import "org/hypertrace/gateway/service/v1/gateway_query.proto";
 
 // Entity + metric data
 message Entity {
+  string id = 1;
```

Comment:
wonder why we started with 2...

Reply:
@buchi-busireddy  told that initially we have added id and removed since that is also an attribute. It is good to have but i can still get entityId from attributesList.

Reply:
Gotcha. Since it's special in both graphql and gateway (i.e. we need to know what its value is and the associated attribute key) we don't gain anything by treating it as a plain attribute and instead need to jump through hoops to extract it. So I think it's a good change, but given that the code to work with the generic id is already in place, we may want to keep it separate from this to keep the change footprint down unless it helps in the implementation - the important part is the id on the baseline response.